readme.md

```diff
@@ -12,22 +13,22 @@
 - [Icon Fonts](#icon-fonts)
 - [Stock Photos](#stock-photos)
 - [Stock Videos](#stock-videos)
-- [Stock Music & Sound Effects](#stock-music--sound-effects)
-- [Vectors & Clip Art](#vectors--clip-art)
-- [Product & Image Mockups](#product--image-mockups)
-- [HTML & CSS Templates](#html--css-templates)
+- [Stock Music \& Sound Effects](#stock-music--sound-effects)
```

Comment:
Please revert this esc slash `Stock Music \&`

Reply:
i've reverted the \& in all file